cloud_sql/postgres_instance_iam_group_auth/main.tf

```
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# [START cloud_sql_postgres_instance_iam_group_auth_create_instance]
+resource "google_sql_database_instance" "default" {
+  name             = "postgres-iam-group-auth-instance-name-test"
```

same as above, not sure we should be having `-test` in our official docs
```suggestion
  name             = "postgres-iam-group-auth-instance-name"
```

---

done